api/src/db-connection.ts

```diff
@@ -114,6 +114,31 @@ export class Connection {
     return { rows, offset, size, total };
   }
 
+  async getPaperEntityCount(paperSelector: PaperSelector): Promise<{paper: PaperSelector, count: number} | null> {
```

Comment:
Stylistic question: why return the paper selector with the count?

Reply:
Not necessarily needed here.